detox/src/ios/expectTwo.js

```diff
@@ -126,6 +126,24 @@ class Element {
     return this.withAction('longPress', duration);
   }
 
+  longPressAndDrag(duration, normalizedPositionX, normalizedPositionY, targetElement,
+            normalizedTargetPositionX, normalizedTargetPositionY, speed = 'fast', holdDuration = 1000) {
```

Comment:
I will change this to have the `normalizedTargetPositionX, normalizedTargetPositionY` as optional, because usually people will want to have `NaN` there anyway.

Reply:
Sure